package.json

```diff
@@ -27,14 +27,15 @@
     "@reach/skip-nav": "0.10.3",
     "@reach/visually-hidden": "0.10.2",
     "algoliasearch": "^4.2.0",
+    "base-64": "^0.1.0",
     "dotenv": "^8.2.0",
     "emotion-theming": "^10.0.27",
     "gatsby": "^2.23.3",
     "gatsby-cypress-endpoints": "0.0.6",
     "gatsby-plugin-chakra-ui": "^0.1.4",
     "gatsby-plugin-create-client-paths": "^2.3.4",
     "gatsby-plugin-facebook-pixel": "^1.0.3",
-    "gatsby-plugin-google-analytics": "^2.3.4",
+    "gatsby-plugin-google-tagmanager": "^2.3.4",
```

Comment:
Not sure where this slipped in from...

Reply:
It'd be from this, but it's going backwards
https://github.com/Rebuild-Black-Business/RBB-Website/pull/198